resources/migrations/001_update_migrations.yaml

```diff
@@ -11557,6 +11464,100 @@ databaseChangeLog:
             constraintName: fk_metabase_database_metabase_database_id
             onDelete: RESTRICT
 
+  - changeSet:
+      id: v55.2025-04-03T10:52:01
```

Comment:
This hoisting is going to get real old :-(

Reply:
Our approach to migrations is so tiresome :(
Even just having migrations in separate files would be better because they would self-order and not conflict in really ugly ways all the time.